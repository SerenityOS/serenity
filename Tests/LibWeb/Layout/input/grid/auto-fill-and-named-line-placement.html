<!DOCTYPE html><html lang="en"><style>
.grid-container {
    display: grid;
    grid-template-columns: [start] repeat(auto-fill, minmax(100px, 1fr)) [end];
}

.grid-item {
    grid-column: start / end;
    background-color: lightblue;
    padding: 20px;
    text-align: center;
}
</style><div class="grid-container"><div class="grid-item">1</div></div>