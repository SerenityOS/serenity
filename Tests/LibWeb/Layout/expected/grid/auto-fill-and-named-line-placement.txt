Viewport <#document> at (0,0) content-size 800x600 children: not-inline
  BlockContainer <html> at (0,0) content-size 800x73.46875 [BFC] children: not-inline
    BlockContainer <body> at (8,8) content-size 784x57.46875 children: not-inline
      Box <div.grid-container> at (8,8) content-size 784x57.46875 [GFC] children: not-inline
        BlockContainer <div.grid-item> at (28,28) content-size 744x17.46875 [BFC] children: inline
          line 0 width: 6.34375, height: 17.46875, bottom: 17.46875, baseline: 13.53125
            frag 0 from TextNode start: 0, length: 1, rect: [397,28 6.34375x17.46875]
              "1"
          TextNode <#text>

ViewportPaintable (Viewport<#document>) [0,0 800x600]
  PaintableWithLines (BlockContainer<HTML>) [0,0 800x73.46875]
    PaintableWithLines (BlockContainer<BODY>) [8,8 784x57.46875]
      PaintableBox (Box<DIV>.grid-container) [8,8 784x57.46875]
        PaintableWithLines (BlockContainer<DIV>.grid-item) [8,8 784x57.46875]
          TextPaintable (TextNode<#text>)
